en/docs/reference/mediators/payloadfactory-mediator.md

```diff
@@ -1047,4 +1047,49 @@ This example shows how to use arguments in a freemarker template to pass values
     </payloadFactory>
     ```
 
-In this example, the value for the “argument one” key is replaced by the first argument value. The argument for the "argument two" key is replaced by the second argument value.
\ No newline at end of file
+In this example, the value for the “argument one” key is replaced by the first argument value. The argument for the "argument two" key is replaced by the second argument value.
+
+### Handling Missing Values
```

Comment:
Shall we say Handling Null/Empty Values? 
The word `Missing` does not give that feeling. 

Also better mention if you know some input values can be optional in the payload, you can construct the FreeMarker template.

Reply:
Modified the description and the title.